shell/demo/demo4.ps1

```diff
@@ -0,0 +1,172 @@
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory=$true, HelpMessage="Prelude Token")]
+    [String]$preludeToken
+)
+
+function FromEnv { param ([string]$envVar, [string]$default)
+    $envVal = [Environment]::GetEnvironmentVariable($envVar, "User")
+    if ($envVal) { return $envVal } else { return $default }
+}
+
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$PRELUDE_API = FromEnv "PRELUDE_API" "https://api.preludesecurity.com"
+
+$TEST_ID="74077d3b-6a2f-49fa-903e-99cad6f34cf6"
+$TEST_INTRO="Will your computer quarantine a malicious mach-o file?
+
+Malicious files are used to gain entry and conduct cyberattacks against corporate systems through seemingly
+innocuous email attachments or direct downloads. For example - a malicious Mac mach-o file was used by Earth Berberoka
```

Comment:
Should we be using `macOS` rather than `Mac`? As far as I can tell, we most commonly use Mac. Just calling to your attention. If there is a strong opinion, lets let it be known.

Reply:
I don't think it matters. Consistency is key though so I'll keep Mac.